js/plugins/checks/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@genkit-ai/checks",
+  "description": "Google Checks AI Safety plugins for classifying the safety of text against Checks AI safety policies.",
+  "keywords": [
+    "genkit",
+    "genkit-plugin",
+    "google cloud",
+    "google ai",
+    "ai",
+    "genai",
+    "generative-ai",
+    "checks",
+    "google checks",
+    "guardrails"
+  ],
+  "version": "0.9.0-dev.2",
+  "type": "commonjs",
+  "scripts": {
+    "check": "tsc",
+    "compile": "tsup-node",
+    "build:clean": "rimraf ./lib",
+    "build": "npm-run-all build:clean check compile",
+    "build:watch": "tsup-node --watch"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/firebase/genkit.git",
+    "directory": "js/plugins/checks"
+  },
+  "author": "genkit",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@google-cloud/aiplatform": "^3.23.0",
```

Comment:
Is `aiplatform` a true dependency for checks?

Reply:
Removed all deps but the google-auth-library